src/Compilers/CSharp/Test/Semantic/Semantics/StackAllocSpanExpressionsTests.cs

```diff
@@ -0,0 +1,488 @@
+        var obj2 = stackalloc int[10];
+        Span<int> obj3 = stackalloc int[10];
+        int* obj4 = stackalloc int[10];
+    }
+    
+    public static implicit operator Test(int* value) 
+    {
+        return default(Test);
+    }
+}", TestOptions.UnsafeReleaseDll);
+
+            comp.VerifyDiagnostics();
+
+            var tree = comp.SyntaxTrees.Single();
+            var model = comp.GetSemanticModel(tree);
+
+            var variables = tree.GetCompilationUnitRoot().DescendantNodes().OfType<VariableDeclaratorSyntax>();
+            Assert.Equal(4, variables.Count());
+
+            var obj1 = variables.ElementAt(0);
+            Assert.Equal("obj1", obj1.Identifier.Text);
+
+            var obj1Value = model.GetSemanticInfoSummary(obj1.Initializer.Value);
+            Assert.Null(obj1Value.Type);
+            Assert.Equal("Test", obj1Value.ConvertedType.Name);
+            Assert.Equal(ConversionKind.StackAllocToPointerType, obj1Value.ImplicitConversion.Kind);
+            Assert.Equal(ConversionKind.ImplicitUserDefined, obj1Value.ImplicitConversion.UnderlyingConversions.Single().Kind);
+
+            var obj2 = variables.ElementAt(1);
+            Assert.Equal("obj2", obj2.Identifier.Text);
+
+            var obj2Value = model.GetSemanticInfoSummary(obj2.Initializer.Value);
+            Assert.Null(obj2Value.Type);
+            Assert.True(obj2Value.ConvertedType is PointerTypeSymbol);
+            Assert.Equal("Int32", ((PointerTypeSymbol)obj2Value.ConvertedType).PointedAtType.Name);
+            Assert.Equal(ConversionKind.StackAllocToPointerType, obj2Value.ImplicitConversion.Kind);
+            Assert.Equal(ConversionKind.Identity, obj2Value.ImplicitConversion.UnderlyingConversions.Single().Kind);
+
+            var obj3 = variables.ElementAt(2);
+            Assert.Equal("obj3", obj3.Identifier.Text);
+
+            var obj3Value = model.GetSemanticInfoSummary(obj3.Initializer.Value);
+            Assert.Null(obj3Value.Type);
+            Assert.Equal("Span", obj3Value.ConvertedType.Name);
+            Assert.Equal(ConversionKind.StackAllocToSpanType, obj3Value.ImplicitConversion.Kind);
+            Assert.Equal(ConversionKind.Identity, obj3Value.ImplicitConversion.UnderlyingConversions.Single().Kind);
+
+            var obj4 = variables.ElementAt(3);
+            Assert.Equal("obj4", obj4.Identifier.Text);
+
+            var obj4Value = model.GetSemanticInfoSummary(obj4.Initializer.Value);
+            Assert.Null(obj4Value.Type);
+            Assert.True(obj4Value.ConvertedType is PointerTypeSymbol);
+            Assert.Equal("Int32", ((PointerTypeSymbol)obj4Value.ConvertedType).PointedAtType.Name);
+            Assert.Equal(ConversionKind.StackAllocToPointerType, obj4Value.ImplicitConversion.Kind);
+            Assert.Equal(ConversionKind.Identity, obj4Value.ImplicitConversion.UnderlyingConversions.Single().Kind);
+        }
+
+        [Fact]
+        public void ConversionFromPointerStackAlloc_UserDefined_Explicit()
```

Comment:
Did you mean `ConversionFromPointerStackAlloc_UserDefined_Implicit` in this file?

---
In reply to: [132531494](https://github.com/dotnet/roslyn/pull/21006#discussion_r132531494) [](ancestors = 132531494)